doc/users/next_whats_new/view_angles_keyword_arguments.rst

```diff
@@ -0,0 +1,17 @@
+Explicit view angle keyword arguments
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It is recommended to provide the 'view angles' for three-dimensional plots
+(with ``mplot3d``) as keyword arguments
+``.view_init(azim=..., elev=..., roll=..., ...)``
+in this order. This is consistent with the order in which the rotations take
```

Comment:
This is outdated. We do not want to change the order. I believe the whole waht's new note should go. Instead put a note on the unconventional order in view_angles.rst and/or the view_init docstring.

Reply:
Removed it.